src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -82,6 +102,31 @@ pub mod tests {
         keyfile_path
     }
 
+    #[allow(dead_code)]
+    pub fn create_resources_file(name: &str) -> PathBuf {
```

Comment:
This should return a result. I think it might also make more sense if it takes a `PathBuf` rather than returning one.

Reply:
updated the codes looks like:
```
    pub fn create_resources_file(resources_file_path: &Path) {
        let resources_file_content = serde_json::json!({
            Policy.to_string(): base64::encode(POLICYJSON.as_bytes()),
            SigstoreConfig.to_string(): base64::encode(SIGSTORECONFIG.as_bytes()),
            GPGPublicKey.to_string(): base64::encode(PUBKEY.as_bytes()),
         });


        fs::write(
            resources_file_path,
            serde_json::to_string(&resources_file_content).unwrap(),
        )
        .unwrap();
    }
```